routes/routes.go

```diff
@@ -27,8 +26,17 @@ type Clients struct {
 
 // Setup registers routes for the service
 func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, c Clients, cacheList cache.List) {
-	log.Info(ctx, "adding routes")
 	r.StrictSlash(true).Path("/health").HandlerFunc(c.HealthCheckHandler)
-	r.StrictSlash(true).Path("/search").Methods("GET").HandlerFunc(handlers.Read(cfg, c.Zebedee, c.Renderer, c.Search, cacheList))
+	r.StrictSlash(true).Path("/search").Methods("GET").HandlerFunc(handlers.Read(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "search"))
+
+	r.StrictSlash(true).Path("/alladhocs").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "all-adhocs"))
+	r.StrictSlash(true).Path("/datalist").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-datalist"))
+	r.StrictSlash(true).Path("/publishedrequests").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "published-requests"))
+	r.StrictSlash(true).Path("/staticlist").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-list"))
+	r.StrictSlash(true).Path("/topicspecificmethodology").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-methodology"))
+	r.StrictSlash(true).Path("/timeseriestool").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "time-series-tool"))
+	r.StrictSlash(true).Path("/publications").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "home-publications"))
+	r.StrictSlash(true).Path("/allmethodologies").Methods("GET").HandlerFunc(handlers.ReadDataAggregation(cfg, c.Zebedee, c.Renderer, c.Search, cacheList, "all-methodologies"))
+
```

Comment:
The handlers all have the same clients passed, have you considered setting up a method with them as a receiver? Similar to in the [filter flex app](https://github.com/ONSdigital/dp-frontend-filter-flex-dataset/blob/develop/handlers/handlers.go) and instantiated on the [routes](https://github.com/ONSdigital/dp-frontend-filter-flex-dataset/blob/613ec8b56e0cbdd31e3d2b4145654aedb5cfb487/routes/routes.go#L35)